README.md

```diff
@@ -117,29 +125,41 @@ optional arguments:
                         Blacklist including serial number of certs to revoke
   --tls-crl-check       Checks SSL/TLS certs against vendor's Certificate
                         Revocation Lists for revoked certificates. CRLs are
-                        found in path specified by --tls-capath. Checks the
+                        found in path specified by --tls_capath. Checks the
```

Comment:
Above you removed `--` but left it here. Please make it consistent.

Reply:
Good catch!  I copied this over from Zabbix I think.  I will fix it there as well